src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/FormMetadata.php

```diff
@@ -105,6 +110,38 @@ public function getSchema(): SchemaMetadata
         return $this->schema;
     }
 
+    /**
+     * @return TagMetadata[]
+     */
+    public function getTags(): array
+    {
+        return $this->tags;
+    }
+
+    public function getTag(string $name): ?TagMetadata
+    {
+        foreach ($this->getTags() as $tag) {
+            if ($tag->getName() === $name) {
+                return $tag;
```

Comment:
Is that really correctly implemented this way? Asking because in Symfony you can also have multiple tags with the same type, wondering how we've implemented it...